thoth/storages/graph/models.py

```diff
@@ -485,7 +485,8 @@ class InspectionRun(Base, BaseExtension):
 
     id = Column(Integer, primary_key=True, autoincrement=True)
 
-    inspection_document_id = Column(Text, nullable=False, unique=True)
+    inspection_document_id = Column(Text, nullable=False)
+    inspection_result_number = Column(Integer, nullable=False)
```

Comment:
The migration will need to state what to do with already existing entries I suppose.

Reply:
I will take a pg dump from stage and verify if the migration works :)

Reply:
Default will be `0` for each inspection document_id, so at least on inspection result was collected